bitcoinspv/bootstrap.go

```diff
@@ -192,8 +192,8 @@ func (r *Relayer) getBTCLatestBlockHeight() (int64, error) {
 	return btcLatestBlockHeight, nil
 }
 
-func (r *Relayer) getNativeLatestBlockHeight() (int64, error) {
-	nativeBlock, err := r.SPVClient.GetLatestBlockInfo(context.Background())
+func (r *Relayer) getNativeLatestBlockHeight(ctx context.Context) (int64, error) {
```

Comment:
Let's rename this function to a more correct context.

Reply:
renamed